### PR TITLE
fix(carbon-components-react) TooltipDefinition missed some props

### PR DESCRIPTION
**Pull request by sgregoire**
Added the missing props `align` and `triggerClassName` from `TooltipDefinition`.

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/TooltipDefinition/TooltipDefinition.js>>

### CODE REVIEW

File: types/carbon-components-react/lib/components/TooltipDefinition/TooltipDefinition.d.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import * as React from "react";
-import { Direction, ReactDivAttr } from "../../../typings/shared";
+import { Direction, ReactDivAttr, TooltipAlignment } from "../../../typings/shared";
 
 interface InheritedProps extends ReactDivAttr { }
 
 export interface TooltipDefinitionProps extends InheritedProps {
     direction?: Extract<Direction, "bottom" | "top">, // required but has default value
+    align?: TooltipAlignment,
+    triggerClassName?: string
```
Comment (kalbert312) [inline types/carbon-components-react/lib/components/TooltipDefinition/TooltipDefinition.d.ts]: It looks like this is not in `carbon-components-react` v.7.6.x so this line will need to be updated: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d56b018b0d6fcbedf0053400c80def5efddd53ab/types/carbon-components-react/index.d.ts#L1 . If it was introduced in v7.7.x, it should be `7.7`, `v8.0.x` would be `8.0` etc.

**Review by kalbert312 — CHANGES_REQUESTED** [review]

File: types/carbon-components-react/lib/components/TooltipDefinition/TooltipDefinition.d.ts (hunk 1 above)
Comment (kalbert312) [reply types/carbon-components-react/lib/components/TooltipDefinition/TooltipDefinition.d.ts]: That line affects the version of the types package published to the registry.

Comment (sgregoire) [reply types/carbon-components-react/lib/components/TooltipDefinition/TooltipDefinition.d.ts]: The `align` prop is there in branch `v10.6.4`, see [TooltipDefinition#78](https://github.com/carbon-design-system/carbon/blob/v10.6.4/packages/react/src/components/TooltipDefinition/TooltipDefinition.js#L78).

You can also see in the [package.json#4](https://github.com/carbon-design-system/carbon/blob/v10.6.4/packages/react/package.json) from the same branch that the version `7.6.0` is still there.

Comment (kalbert312) [reply types/carbon-components-react/lib/components/TooltipDefinition/TooltipDefinition.d.ts]: Looks like I mixed myself up.

**Review by kalbert312 — APPROVED** [review]

**Pull request merged**